### PR TITLE
would it be possible for you to merge my pull request. We are now using python 3.9 and like to use the rabbitmq-api-admin

### PR DESCRIPTION
**Pull request by vlogic**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -11,7 +11,7 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = "~3.8"
+python = "~3.9"
```
Comment (PetrS12) [inline pyproject.toml]: I think this will break projects written in python 3.8. Better to leave it unchanged

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -5,7 +5,7 @@ on: [push, pull_request]
 jobs:
   autotests:
     runs-on: ubuntu-latest
-    container: python:3.8
+    container: python:3.9
```
Comment (PetrS12) [inline .github/workflows/tests.yml]: It is better to use github matrix here for testing with different versions of python(3.8, 3.9)
https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix

File: pyproject.toml (hunk 1 above)
Comment (vlogic) [reply pyproject.toml]: ok

**Comment by gaby** [comment]
@PetrS12 Can we get this merged?

**Comment by PetrS12** [comment]
@vlogic, @gaby , thanks for your PR. Moved in other PR and merged all the changes. Released the new tag 0.2.0.

**Comment by gaby** [comment]
@PetrS12 Thank youu!

**Pull request closed without merging**